src/test/java/bio/terra/integration/SamFixtures.java

```diff
@@ -82,6 +84,27 @@ public void deleteServiceAccountFromTerra(TestConfiguration.User user, String se
     }
   }
 
+  public void deleteSnapshotAccessRequest(
+      TestConfiguration.User steward, UUID snapshotAccessRequestId) {
```

Comment:
it's possible the user isn't `steward` here
```suggestion
      TestConfiguration.User user, UUID snapshotAccessRequestId) {
```